pkg/exporter/sumologicexporter/README.md

```diff
@@ -59,6 +59,13 @@ exporters:
       - <regex1>
       - <regex2>
 
+    # instructs sumologicextension to use automatically generated sumologic endpoint;
+    # to use direct endpoint, set it to null
+    # see sumologicextension documentation for details
+    # default = "sumologic"
+    auth:
+      authenticator: {"sumologic", null}
```

Comment:
Actually can we change to to not include quotation marks as other defaults in comments here? They are not required for yaml so let's not add complexity where not needed.

Reply:
meh, auto merge :disappointed:

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/pull/85